lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -79,14 +82,32 @@ def mulled_tags_for(namespace, image, tag_prefix=None):
 
 def split_tag(tag):
     """Split mulled image name into conda version and conda build."""
-    version = tag.split('--', 1)[0]
-    build = tag.split('--', 1)[1]
-    return version, build
+    return tag.split('--', 1)
+
+
+def parse_tag(tag):
+    """Split mulled image name into version and build number."""
+    version = tag
+    build_string = "-1"
+    if '--' in tag:
+        version, build_string = tag.rsplit('--', 1)
```

Comment:
isn't that what split_tag() should do?

Reply:
`split_tag` is used on image names, so this wouldn't return the correct elements.